getting-started/run-queries-from-console.md

```diff
@@ -138,11 +175,15 @@ If you'd like to prevent write operations such as insert or update. Instead
 of using the `tsdbadmin` superuser, create a read-only user for your service and
 use that within PopSQL.
 
-### How does pricing work?
+#### How does pricing work?
 
 Timescale customers get PopSQL Business for free for up to 10 users. To add more than 10 seats to your account, [Contact us](https://www.timescale.com/contact) for further assistance.  
```

Comment:
Aside: but Timescale/PopSQL pricing has changed, and we should update this. See Timescale's pricing page for details

Reply:
I've moved this to a top level section and pointed to the pricing tier feature list.